Features/tijdreizen.feature

```diff
@@ -0,0 +1,111 @@
+    Dan bevat de response het voorkomen van een object dat geldig is op het moment van bevragen
+    En dat geregistreerd staat op het moment van bevragen
+  
+  Scenario: geldigOp niet en beschikbaarOp wel opgegeven in request
+    Als bij een endpoint geldigOp en beschikbaarOp ondersteund worden
+    En geldigOp is niet opgegeven in het request
+    En beschikbaarOp is wel opgegeven in het request
+    Dan bevat de response het voorkomen van een object dat geregistreerd stond op het opgegeven moment beschikbaarOp
+    En dat met de kennis van toen geldig is op het moment van bevragen
+  
+  Scenario: geldigOp wel en beschikbaarOp niet opgegeven in request
+    Als bij een endpoint geldigOp en beschikbaarOp ondersteund worden
+    En geldigOp is wel opgegeven in het request
+    En beschikbaarOp is niet opgegeven in het request
+    Dan bevat de response het voorkomen van een object dat op dit moment geregistreerd staat
+    En dat met de kennis van dit moment, geldig was op het moment geldigOp
+  
+  Scenario: geldigOp en beschikbaarOp opgegeven in request
+    Als bij een endpoint geldigOp en beschikbaarOp ondersteund worden
+    En geldigOp is wel opgegeven in het request
+    En beschikbaarOp is wel opgegeven in het request
+    Dan bevat de response het voorkomen van een object dat geregistreerd stond op het opgegeven moment beschikbaarOp
+    En dat met de kennis van toen, geldig was op het moment geldigOp
+  
+  Scenario: endpoint ondersteunt query parameters geldigOp en beschikbaarOp niet
+    Als bij een endpoint geldigOp en beschikbaarOp niet wordt ondersteund
+    En in een request wordt geldigOp en/of beschikbaarOp opgegeven
+    Dan worden de geldigOp en/of beschikbaarOp parameters genegeerd
+
+  Scenario: query parameter huidig in request, status object voorkomen is geen eindstatus
+    Als bij een endpoint geldigOp, beschikbaarOp en huidig worden ondersteund
+    En in een request wordt geen geldigOp opgegeven
+    En in de request wordt geen beschikbaarOp opgegeven
+    En in de request wordt huidig=true opgegeven
+    En de status van het betreffende object voorkomen is geen eindstatus
+    Dan bevat de response het actuele voorkomen van een object
+
+  Scenario: query parameter huidig in request, status object voorkomen is eindstatus
+    Als bij een endpoint geldigOp, beschikbaarOp en huidig worden ondersteund
+    En in een request wordt geen geldigOp opgegeven
+    En in de request wordt geen beschikbaarOp opgegeven
+    En in de request wordt huidig=true opgegeven
+    En de status van het betreffende object voorkomen is een eindstatus
+    Dan bevat de response geen resultaat
+
+  Scenario: query parameter geldigOp, beschikbaarOp en huidig in request
+    Als bij een endpoint geldigOp, beschikbaarOp en huidig worden ondersteund
+    En in een request wordt geldigOp opgegeven
+    En in de request wordt beschikbaarOp opgegeven
+    En in de request wordt huidig=true opgegeven
```

Comment:
Hier is nu specifiek `huidig=true` opgenomen. Klopt de volgende aanname dan?
"Als huidig=false is opgegeven en geldigOp en/of beschikbaarOp wordt opgegeven, dan krijg je geen validatiemelding."

Wat gebeurt er als je `huidig=true` opgeeft en `geldigOp=vandaag`?